online-shopping-website/frontend/src/Components/CartPage.js

```diff
@@ -63,14 +75,14 @@ export const CartPage = () => {
         });
 
         //Clearing cart
-        cart = [];    //TODO: replace with delete cart cookie (on alert open)
+        cartCookie.cart = [];    //TODO: replace with delete cart cookie (on alert open)
```

Comment:
Should we finish deleting cart cookies in this PR or a future one?
Maybe this quick fix would be nice, should be one line, look up how to delete a cookie with react cookie